modules/storages/spec/common/storages/peripherals/nextcloud_connection_validator_spec.rb

```diff
@@ -230,6 +230,25 @@
     end
   end
 
+  context "when OpenID Connect is available" do
```

Comment:
:yellow_circle: Context name sounds misleading to me. From what I read, OpenID Connect (aka _SSO_ Authentication) is **enabled**, but there is no provider **available**.

```suggestion
  context "when OpenID Connect is enabled" do
```

Reply:
Oh, good catch!